tests/SparqlLuaTest.php

```diff
@@ -0,0 +1,33 @@
+<?php
+
+use MediaWiki\Http\HttpRequestFactory;
+use PHPUnit\Framework\TestCase;
+use ProfessionalWiki\SPARQL\SparqlLuaQueryRunner;
+
+class SparqlLuaTest extends TestCase {
```

Comment:
Don't forget to namespace the test. You can look at our other extensions to see what we do.

Also should have a `@covers` tag (as per coding conventions in our company handbook)

Reply:
I noticed you put a @covers tag on the method level instead. I suppose that also works.

`SparqlLuaTest` should be renamed to `SparqlLuaQueryRunnerTest`. I've now documented this in our coding conventions.